app/frontend/src/service/user.ts

```diff
@@ -49,7 +49,8 @@ export type UpdateUserProfileData = Pick<
   | "countriesVisited"
   | "countriesLived"
   | "additionalInformation"
->;
+  | "avatarKey"
```

Comment:
You need to apply the new data to the gRPC request message in this file in `updateProfile` function